unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BotsController/BotsController.cs

```diff
@@ -0,0 +1,291 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL.Controllers;
+using DCL.Interface;
+using DCL.Models;
+using Google.Protobuf;
+using UnityEngine;
+using DCL.Configuration;
+using UnityEngine.Networking;
+using Random = UnityEngine.Random;
+
+namespace DCL.Bots
+{
+    /// <summary>
+    /// Bots Tool: BotsController
+    ///
+    /// Used to spawn bots/avatarShapes for debugging and profiling purposes.
+    /// </summary>
+    public class BotsController : IBotsController
+    {
+        private const string ALL_WEARABLES_FETCH_BASE_URL = "https://peer.decentraland.org/content/deployments";
+        private const string ALL_WEARABLES_FETCH_FIRST_URL = ALL_WEARABLES_FETCH_BASE_URL + "?entityType=wearable&onlyCurrentlyPointed=true";
+
+        private ParcelScene globalScene;
+        private List<string> instantiatedBots = new List<string>();
+        private List<string> eyesWearableIds = new List<string>();
+        private List<string> eyebrowsWearableIds = new List<string>();
+        private List<string> mouthWearableIds = new List<string>();
+        private List<string> hairWearableIds = new List<string>();
+        private List<string> facialWearableIds = new List<string>();
+        private List<string> upperBodyWearableIds = new List<string>();
+        private List<string> lowerBodyWearableIds = new List<string>();
+        private List<string> feetWearableIds = new List<string>();
+        private List<string> bodyshapeWearableIds = new List<string>();
+
+        /// <summary>
+        /// Makes sure the Catalogue with all the wearables has already been loaded, otherwise it loads it
+        /// </summary>
+        private IEnumerator EnsureGlobalSceneAndCatalog()
+        {
+            if (globalScene != null)
+                yield break;
+
+            globalScene = Environment.i.world.state.loadedScenes[Environment.i.world.state.globalSceneIds[0]] as ParcelScene;
+
+            CatalogController.wearableCatalog.Clear();
+
+            yield return GetAllWearableItems(ALL_WEARABLES_FETCH_FIRST_URL);
```

Comment:
Do we really need the whole catalog? This won't scale very well with the new wearables editor.

Reply:
True, I have to refactor and change how we fetch the collections for the catalogue because o an API change so I'll also change it to randomize the collections it'll download and use the randomized collections locally to populate the catalogue (instead of populating it with ALL the collections)

Reply:
Done!